nodeup/pkg/bootstrap/install.go

```diff
@@ -153,6 +153,14 @@ func (i *Installation) buildEnvFile() *nodetasks.File {
 		envVars["AZURE_STORAGE_ACCOUNT"] = os.Getenv("AZURE_STORAGE_ACCOUNT")
 	}
 
+	if os.Getenv("SCW_ACCESS_KEY") != "" {
```

Comment:
Should it be secret key? It is always required to have a secret key but not always to have an access key